web/js/component/code_block.js

```diff
@@ -23,13 +23,10 @@ import {isDefinedAndNotNull, replaceAll} from "../utils";
 // since that file is a custom build that includes only the languages we use.
 const LANGUAGE_PROP_TYPE = PropTypes.oneOf([
   "java",
-  "r",
   "python",
   "bash",
   "json",
-  "matlab",
   "yaml",
-  "dockerfile",
   "plaintext",
```

Comment:
Why dockerfile? Is that for orchestrate?

Reply:
We had dockerfile examples for orchestrate back when we hosted the orchestrate docs at app.sigopt.com/docs

Reply:
If that's already gone then great.